homeassistant/components/apple_tv/remote.py

```diff
@@ -48,7 +49,17 @@ async def async_send_command(self, command, **kwargs):
             return
 
         for single_command in command:
-            if not hasattr(self.atv.remote_control, single_command):
+            _LOGGER.info("Sending command %s", single_command)
+            if single_command.startswith("delay="):
```

Comment:
The remote send command service already supports a delay parameter.

https://github.com/home-assistant/core/blob/5ce49c62b143a4b0d9913687e9a37a8e9fdee35d/homeassistant/components/remote/__init__.py#L98

Reply:
How would that be accessed at the script level?

Reply:
Do you mean in the script configuration?

Reply:
It's okay. Thanks. Reviewed a couple of the other remote devices and figured out what I need to do.
Now changed to use the `delay_secs` attribute and also added in the `num_repeats` attribute too.